books/RayTracingTheNextWeek.html

```diff
@@ -39,28 +39,27 @@
 
 Motion Blur
 ====================================================================================================
-When you decided to ray trace, you decided that visual quality was worth more than run-time. In your
-fuzzy reflection and defocus blur you needed multiple samples per pixel. Once you have taken a step
-down that road, the good news is that almost all effects can be brute-forced. Motion blur is
-certainly one of those. In a real camera, the shutter opens and stays open for a time interval, and
-the camera and objects may move during that time. Its really an average of what the camera sees over
-that interval that we want.
+When you decided to ray trace, you decided that visual quality was worth more than run-time. When
+rendering fuzzy reflection and defocus blur, we used multiple samples per pixel. Once you have taken
+a step down that road, the good news is that almost _all_ effects can be similarly brute-forced.
+Motion blur is certainly one of those.
+
+In a real camera, the shutter remains open for a short time interval, during which the camera and
+objects in the world may move. To accurately reproduce such a camera shot, we seek an average of
+all the instant images that the camera perceives while its shutter is open to the world.
```

Comment:
"an average of all the instant images"
vs
"an average of what the camera sees over the interval"

Your version is more explicit. And more explicit is usually more good.
But I prefer Shirley's because it allows the reader to come up with their own mental model for how motion blur works.

Your text is explicit that a camera shot is the integral of infinitesimal images captured over the shutter period.
My (different) mental model is that a camera shot is the integral of each individual photon over the shutter period.

Both are equally valid, but your text is choosing the mental model for me :(

Reply:
So I guess what I'm trying to say is that your revision is worse by being _better_

Reply:
I think it's important to distinguish between the reality and the model. I agree with your take; will fix.